src/contracts/digital-twin/digital-twin.ts

```diff
@@ -575,6 +582,22 @@ export class DigitalTwin extends Logger {
     );
   }
 
+  private async createAndStoreDidDocument(twinIdentityId: string, controllerId: string):
+  Promise<void> {
+    const twinDid = await this.options.did.convertIdentityToDid(twinIdentityId);
+    const controllerDid = await this.options.did.convertIdentityToDid(controllerId);
+    if (!controllerDid) {
```

Comment:
We can remove the `if (!controllerDid) {` section as it will not return a falsy value.